controller/backup_controller.go

```diff
@@ -32,6 +32,14 @@ import (
 	longhorn "github.com/longhorn/longhorn-manager/k8s/pkg/apis/longhorn/v1beta2"
 )
 
+var (
+	// maxRetriesOnAcquireLockError should guarantee the cumulative retry time
+	// is larger than 150 seconds.
+	//
+	// 5ms, 10ms, 20ms, ... , 81.92s, 163.84s
```

Comment:
Could you add the math function on how it is calculated? `(5ms*2^(maxRetries-1))`

Reply:
Okay. added.